src/platform_impl/android/mod.rs

```diff
@@ -1012,6 +1012,10 @@ impl MonitorHandle {
         self.app.config().density().map(|dpi| dpi as f64 / 160.0).unwrap_or(1.0)
     }
 
+    pub fn bit_depth(&self) -> u16 {
+        32
```

Comment:
On Android this seems hard. We could ask for the current format but various APIs can always change it.

I've seen though that after changing the Surface/window format, APIs like EGL suddenly return just the configs matching that exact format.

Reply:
> We could ask for the current format but various APIs can always change it.

I think it should be clear to users that the data only reflects the state at the time it was queried.

In the future it would be nice if we could have some events that reflect a change in available monitors or their settings. Its definitely on my checklist for Web.

> On Android this seems hard.

I'm not too worried about it in this PR because `VideoModeHandle` on Android already always returns 32, so this definitely doesn't change anything about the current state of things.

Reply:
We solved this by returning `None` here now.
See https://github.com/rust-windowing/winit/pull/3802#discussion_r1690481503 for more context.